### PR TITLE
Make moonbeams available

### PR DESCRIPTION
**Pull request by warp-core**
**Content (Artwork / Missions / Jobs)**

## Summary
This PR changes a number of things at the current end of the Wanderer storyline:
- Added variants of the Hurricane, Derecho, and Tempest which use Moonbeams and Moonbeam Turrets.
- Adds some more ships to the `Wanderers: Sestor: Factory Escorts fleet, because the final fight seems extraordinarily difficult. These additional ships carry Moonbeams and Moonbeam Turrets.
- Added some dialog to "Wanderers: Sestor: Factory 1" discussing the Moonbeams.
- Added a (somewhat) unique sound for the Moonbeam (it's just the Sunbeam sound with the pitch reduced). (It could do with some more work, but it makes the Moonbeam sound different to the Sunbeam now.)
Thanks to @Saugia who improved the sound.
- Added a Small Biochemical Cell to the stock Summer Leaf. The ship has a lot of free space (52) and, with the recent buff, the SBC more than doubles the energy capacity on the ship, it makes sense to have this here since the ship primarily ffights the Unfettered Hai.
- ~The Exiles have drones event now only adds Model 16's, 32's, Far Osk 27's, Far Lek 14's, MPT 53's, and TF 109's to the Korath Home fleet. The reasoning is that beginning to produce large automata so soon is an abrupt increase in manufacturing capacity that probably shouldn't happen. (@ravenshining agrees.) Increases in the Exiles' capacity to build automata should come over time (probably triggered by future missions).~
- ~The Exiles have drones event is split into four events each adding larger ships:~
  - ~The first event adds 53s, 16s, 32s, and 14s and 27s flying around ontheir own (every where with Korath Home should have a station or something for them to launch from). 20 days after the final mission.~
  - ~The second event removes the lone intrasystem ships and adds 109s carrying them as well as 32s, 64s, and 128s. 80 days after the final mission.~
  - ~The third event adds 71s, 78s, and 256s. 160 days after the final mission.~
  - ~The fourth and final event adds 512s and 349s. 250 days after the final mission.~
- ~Added an invisible patch mission that will remove the larger automata from the Korath Home fleet on saves that already have them.~
  - ~The above will make it impossible to capture TF 71's, TF 78's, and KIV 349's, and all but the Mereti light warships (without doing irreversible reputation damage for the Mereti ships) but ships without Jump Drives captured in Exile space were stuck there anyway. The full range of outfits is still available, I think.~
- Renamed "Wanderers: Sestor: Final" to "Wanderers: Sestor: Final: Patched" and moved some event calls from patch missions into it. Those patch missions will fail if this renamed mission is offered, so they are still available to saves where the original version of the mission was offered but no longer necessary for new saves.
- Added Moonbeam variant ships to the Wanderers Kor Efreti Traders fleet.
I think that's everything.

## Save File
This save file can be used to play through the new mission content:
[WarpCoreWandererSestor MissionChangeTester~3014-11-16.txt](https://github.com/endless-sky/endless-sky/files/9302930/WarpCoreWandererSestor.MissionChangeTester.3014-11-16.txt)
Take off and land to be offered the mission.

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - ~I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}~
 - ~I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~

### CODE REVIEW

**Comment by ravenshining** [comment]
As mentioned on Discord, I think the Tek Far 109 is a bit too big for the Exiles to begin producing right from the get-go.  If they need to be carried, Far Osk 27s can be carried by Raiders & Dredgers, and once #7088 is merged, Far Lek 14s by Ra'at Ik 621s.

Personally I'd set up a timed event, rather than waiting on new wanderer missions to come along.  Have the Sestor carriers & up to Model 128 come after a several months, then the KIV and up to Model 512 after several more.

Another tweak I might do while you're here could be to have Exile automata be variants with pre-installed command centres.

**Comment by Zitchas** [comment]
We should probably have a conversation about how to handle the Korath Exile progression. I've got a large mission set in the works for the Remnant response to the Exiles gaining automata tech, and obviously there's a possibility of the Wanderer story also being involved in it... But the Exiles aren't really a central feature in the Wanderer story, it's more focused on the Efreti/Mereti and the Northern area. This isn't the place for that, though.

So I'll just say that for now, I like Ravenshining's idea that the Exiles should start out with minimal Automata tech, and have it expand over time as their industrial capacity increases.

**Comment by Hurleveur** [comment]
> Another tweak I might do while you're here could be to have Exile automata be variants with pre-installed command centres.

I am all for the idea of them not trusting this AI that betrayed them anymore.
However, currently we can't really install more than one command centre, and one will not be enough for a ship with turrets.
That means that for the big boys (KIV & donut) it will not be sufficient, so we would need an improved version of the command centre if we want to do that, one that would provide 5 crew or smth of the like, or we need to remove the repeater turrets from them.

If we just give them command centers they will act eratically because of the lack of crew.

**Comment by ravenshining** [comment]
That's what bunk rooms are for.

**Comment by warp-core** [comment]
Adjusted the Exiles have drones event:
The first event adds 53s, 16s, 32s, and 14s and 27s flying around ontheir own (every where with Korath Home should have a station or something for them to launch from). 20 days after the final mission.
The second event removes the lone intrasystem ships and adds 109s carrying them as well as 32s, 64s, and 128s. 80 days after the final mission.
The third event adds 71s, 78s, and 256s. 160 days after the final mission.
The fourth and final event adds 512s and 349s. 250 days after the final mission.

**Comment by Zitchas** [comment]
That seems like a reasonable time line. Once we have missions sets that tie in with this; can we plan on removing the timers and just having the missions trigger the relevant events when appropriate?

File: data/wanderer/wanderers middle.txt
Comment (ravenshining) [reply data/wanderer/wanderers middle.txt:3135]: Hm...  almost there.  How about alluding to their agrarian tendencies with:

```suggestion
			`	Tele'ek seems pleased that you noticed. "Indeed. They are more [useful, effective] against hull armor than shields, [perfect, appropriate] for use against the Kor Sestor vessels alongside our [existing, previous] armaments." He pauses before continuing. "[Hopefully, If this goes well], they will not get much use, but will see you [safely, successfully] through this [mission, task] to finally end this [blight, menace]. However, we will have them [ready, prepared] if the need does [arise, return]."`
```

**Comment by warp-core** [comment]
> That seems like a reasonable time line. Once we have missions sets that tie in with this; can we plan on removing the timers and just having the missions trigger the relevant events when appropriate?

The idea behind the timed events is that it doesn't make sense for them to immediately start building these massive ships that are significantly different and more advanced than the ships they've been building up until this point, there should be a delay in which they build up their capacity and retool production facilities, etc. Unless these missions would involve them getting a boost in that area to justify a faster rollout, I'm not sure that would make sense, but it is certainly possible to do.

Do note, however, that I'm not sure how triggering an event that's already been set on a timer works. Events aren't one time things, if a mission calls an event that's already on a timer from another mission, it will likely end up being applied a second time after the timer has elapsed. This is something that will need to be dealt with if you want to accelerate the rollout.

**Comment by Zitchas** [comment]
I wasn't really thinking of accelerating the rollout, but rather having the timing tied to the story. Story progression won't be quite as predictable as timers, but we can be fairly confident (based on travel times and whatnot), that a certain number of missions will take a certain length of time.

Ex. If we have a long series of missions doing recon on the Exiles industry and development process, and over the course of the missions new stuff starts showing up. It's more about having a story to explain what the events are doing more so than any real change to the timing itself.

**Comment by warp-core** [comment]
> I wasn't really thinking of accelerating the rollout, but rather having the timing tied to the story. Story progression won't be quite as predictable as timers, but we can be fairly confident (based on travel times and whatnot), that a certain number of missions will take a certain length of time.

We could change the timers in future to better match up with any mission chains that are involved, yes. However, anyone who does the mission with the currently present timers would still have the events on those timers.

If the preference is that the gradual rollout should be reflected in missions which may not follow the timeline present in this PR, then I'd prefer to not make any changes to the rollout here and leave that to any such missions.

**Comment by Amazinite** [comment]
Similar to #6809, I am assigning myself to this PR as I see this as on step in a larger Wanderer campaign rework that I believe should be done in the future. This PR also acts as a first step in the continuation of the Wanderer campaign past what we already have, which I'd also like to keep an eye on.

**Review by tibetiroka — COMMENTED** [review]
If Rek is speaking Hai on the marked line then it should be fine; otherwise I may have some issue with the wording there. Rest of the grammar should be fine though.

File: data/wanderer/wanderers middle.txt
Comment (warp-core) [reply data/wanderer/wanderers middle.txt:3142]: Yes. Tele'ek doesn't know the human language and we have our translator so she speaks Hai here.

**Review by tibetiroka — APPROVED** [review]
Grammar is OK.

**Review by TheMarksman-ES — APPROVED** [review]

**Review by Hurleveur — APPROVED** [review]
I really like it, the campaign isnt going to continue anytime soon, and when it does we can always make new outfits for it, as that's the most engaging part of content I think.
Do note this isn't my area of expertise though I'm just giving my opinion.

File: data/wanderer/wanderers middle.txt
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:3324]: We have like four patch missions down here now. Can we not clean some of these up?

File: data/wanderer/wanderers middle.txt
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:3073]: I'd prefer this to be a response and a goodbye instead of an action choice. You really just going to walk away from Tele'ek without saying goodbye?

File: data/wanderer/wanderers middle.txt
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:3066]: This should be some sort of affirmation that you're accepting the mission, something that makes sense to jump to the end without hearing more about the weapon, instead of just nothing.

File: data/wanderer/wanderers middle.txt
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:3076]: The transition of "Tele'ek gestures to Rek to speak, who looks back to Tele'ek before he continues," feels clunky to me. I'd just trim this down.

Also seems like we're missing a word in the middle here.
```suggestion
			`	Tele'ek explains, "If they view these weapons as signs of [aggression?] against themselves, there is little I can do for them. Their ships are not as [durable, well-armored] as the Kor Sestor automata, so our other technologies would already be [effective, damaging] enough if used against them. If they are [rational, intelligent], they will value the added protection on this [journey, endeavor]."`
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: data/wanderer/wanderers middle.txt
Comment (Amazinite) [inline data/wanderer/wanderers middle.txt:3376]:
```suggestion
# This patch mission is necessary for people who got the older version of the "Sestor: Final" mission without the patches integrated.
```

**Pull request merged**
